README.md

```diff
@@ -2,11 +2,44 @@
 
 [![Documentation][badge-docs]][link-docs]
 
-This repository hosts tutorials for performing data analyses with [scverse tools](https://scverse.org).
-The tutorials found here are _shared tutorials_ that complement more specific tutorials provided by invidiual
-[core](https://scverse.org/packages/) and [ecosystem](https://scverse.org/packages/#ecosystem) packages.
+On [scverse.org/learn](https://scverse.org/learn), we aim at providing a comprehensive overview of analyses that can be
+performed with scverse core and ecosystem packages.
 
-Please check out the [learn](https://scverse.org/learn) page on scverse.org for an overview of all tutorials!
+To this end, this repository contains
+
+-   a registry for tutorials listed on [scverse.org/learn](https://scverse.org/learn) (see `tutorial-registry`)
+-   shared tutorials that complement more specific tutorials provided by invidiual [core](https://scverse.org/packages/)
+    and [ecosystem](https://scverse.org/packages/#ecosystem) packages (see `docs`)
+
+## Adding tutorials
+
+If you believe a tutorial should be added to `scverse.org/learn`, please open an issue. We will discuss the request
+in the next [open community meeting](https://hackmd.io/VfVLKb3ETGKN2j_7tn8ZJQ?view).
+
+For being added to our website, tutorials must fulfil at least the following requirements:
+
+-   all featured packages must be scverse [core](https://scverse.org/packages/#core-packages) or
+    [approved ecosystem packages](https://scverse.org/packages/#ecosystem). This does not apply to packages that are not
+    specific to omics data analysis (e.g. pandas, seaborn).
+-   the notebook author agrees to maintain the tutorial in the future and is reachable via [zulip](https://scverse.zulipchat.com).
+-   the notebook contains a backlink to [scverse.org/learn](scverse.org/learn)
+
+## Structure of external tutorials
+
+While we do not mandate a specific structure for tutorials,
+a good tutorial typically comprises the following sections:
+
+1. **General header**: The tutorial should have a general header that corresponds to the analysis.
+2. **Brief introduction**: The tutorial should introduce the package, the analysis motivation and potentially biological background.
+3. **Requirements to run the notebook**: Special computational requirements like memory or GPUs should be specified. Any required input from other notebooks should also be listed here.
```

Comment:
I'd like to mention somewhere that these tutorials should be somewhat self contained and not require weird input from some random places. Not sure whether this is the best place to enforce/mention this.